hooks/honeybadger/honeybadger.go

```diff
@@ -0,0 +1,36 @@
+package logrus_honeybadger
+
+import (
+	"github.com/DavidHuie/gobadger"
+	"github.com/Sirupsen/logrus"
+)
+
+// HoneybadgerHook to send exceptions to an exception-tracking service compatible
+// with the Honeybadger API.
+type HoneybadgerHook struct {
+	conn *gobadger.Conn
+}
+
+func NewHoneyBadgerHook(apiKey string) *HoneybadgerHook {
+	return &HoneybadgerHook{gobadger.NewConn(apiKey)}
+}
+
+func (hook *HoneybadgerHook) Fire(entry *logrus.Entry) error {
+	err := hook.conn.Error(entry.Message)
```

Comment:
Hi @Sirupsen,

mainly I was following the implementation of the airbrake hook: https://github.com/Sirupsen/logrus/blob/master/hooks/airbrake/airbrake.go#L36

TBH, if we want to return an error there we will to need to wait always for the response and check it. Perhaps it could be a configuration on this hook to launch the notification async:

```
go hook.coon.Error(entry.Message)
```

But this will raise some other issues than can or can not be important (for example, what happens if it errors? and if the project goes out before returning from the subroutine?).

If your main concern is more: what happens if it hangs that what happens if it is slow, I suspect that we could add some timeout.

Thanks for the feedback! It's a good point, but I don't really know how to approach it.